lib/srv/desktop/windows_server.go

```diff
@@ -919,6 +920,11 @@ func (s *WindowsService) connectRDP(ctx context.Context, log *slog.Logger, tdpCo
 	}
 	createUsers := err == nil
 
+	layout, err := s.cfg.AuthClient.GetKeyboardLayout(ctx, &userpreferencesv1.GetKeyboardLayoutRequest{Username: identity.Username})
+	if err != nil {
+		return trace.Wrap(err)
+	}
```

Comment:
I wonder if we can/should try to do some caching to avoid increasing the latency when starting a session.

@rosstimothy WDYT?